### ISSUE TITLE
Spritesheet usage unclear

### ISSUE THREAD

**Issue by achuinard** [inlet/react-pixi#270]
I am somewhat new to Pixi so apologies if this is a basic question, but I am unsure what the best way to include spritesheets. It'd be awesome if the `<Stage>` component took a `spritesheets` prop (i.e. ['assets/spritesheet1.json', 'assets/spritesheet2.json'] and then all of those textures were immediately available for use. Right now, it seems I have to do some work in onMount and onUnmount, then store the sprites in a state variable, and make sure none of my child components (children to `<Stage>`) render until the spritesheet is loaded.

**Comment by inlet** [comment]
Hi Anthony,

Textures is a topic on its own, they can be interchanged to multiple components, cached, reused and preloaded in multiple ways. For this reason it isn't implemented in this lib as it cannot be used in a standardized React way (load, unload/cleanup in reconciliation).

Anyways, it's quite easy to create a spritesheet loader that fit your needs, here are examples:

https://codepen.io/inlet/pen/wvWRLGL
https://codepen.io/inlet/pen/ewdXEY

Basically it comes down to something like: 

```jsx
<Textures sprite="./my-sprite.json">
  {textures => (
    <AnimatedSprite textures={textures} />
  )}
</Textures>
```

Hopes this helps you, btw did you know that you can always ask these kind of questions in our Slack channel.

Thanks!
